jqassistant.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.7.2
-publicVersions=1.7.3
+archivedVersions=1.7.3
+publicVersions=1.8.1
  
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.8.1.description=Upgraded to jQAssistant 1.8.0 
+1.8.1.date=2002-02-03
+1.8.1.sqVersions=[7.9,LATEST]
+1.8.1.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#181
+1.8.1.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.8.1/sonar-jqassistant-plugin-1.8.1.jar
+
 1.7.3.description=Clarified behavior of configuration property `sonar.jqassistant.reportPath` and improved message for reported issues
 1.7.3.date=2019-12-19
-1.7.3.sqVersions=[7.9,LATEST]
+1.7.3.sqVersions=[7.9,7.9*]
```

Comment:
You want to set this end range to 8.1. That way the Marketplace can suggest your new version to 8.1 users who are currently on this version of your plugin. Otherwise the Marketplace can't make the connection.

Reply:
Done, applied the same for 1.7.2 (hope that's right?)